appcelerator-archive/amp
# Vendoring update

Fix #707 

- updated vendors
- renamed packaged name for nats
- gotools 1.3.0

to test:
```
$ docker volume rm glide
$ rm -rf vendor
$ make -f Makefile.refactor.make install-deps     # this should bring back the workspace in a clean state
$ make -f Makefile.refactor.make update-deps      # this should leave the workspace untouched
$ make clean install
$ make TAG=local build-image
$ amp pf pull && amp pf start --local && make test
```
